lib/mojito/pool.ex

```diff
@@ -1,151 +1,125 @@
 defmodule Mojito.Pool do
-  @moduledoc ~S"""
-  Mojito.Pool provides an HTTP request connection pool based on
-  Mojito and Poolboy.
-
-  Example:
-
-      >>>> children = [Mojito.Pool.child_spec(MyPool)]
-      >>>> {:ok, _pid} = Supervisor.start_link(children, strategy: :one_for_one)
-      >>>> Mojito.Pool.request(MyPool, :get, "http://example.com")
-      {:ok, %Mojito.Response{...}}
-  """
-
-  alias Mojito.Utils
-
-  defp pool_opts, do: Application.get_env(:mojito, :pool_opts, [])
+  @moduledoc false
+
+  ## Mojito.Pool is an HTTP client with high-performance, easy-to-use
+  ## connection pools.
+  ##
+  ## Pools are maintained automatically by Mojito, requests are matched to
+  ## the correct pool without user intervention, and multiple pools can be
+  ## used for the same destination in order to reduce concurrency bottlenecks.
+  ##
+  ## `Mojito.Pool.request/1` is intended for use through `Mojito.request/1`.
+  ## Config parameters are explained in the `Mojito` moduledocs.
+
+  alias Mojito.{Request, Utils}
+  require Logger
+
+  @type pool_opts :: [pool_opt | {:destinations, [pool_opt]}]
+
+  @type pool_opt ::
+          {:size, pos_integer}
+          | {:max_overflow, non_neg_integer}
+          | {:pools, pos_integer}
+          | {:strategy, :lifo | :fifo}
+
+  @typep pool_key :: {String.t(), pos_integer}
+
+  @default_pool_opts [
+    size: 5,
+    max_overflow: 10,
+    pools: 5,
+    strategy: :lifo,
+    refractory_period: 3000
```

Comment:
This key seems wrong :thinking:

Reply:
Good eyes! That key did something (or tried to 😄) at some point in development.